lib/auth/access.go

```diff
@@ -111,7 +117,7 @@ func (a *Server) DeleteRole(ctx context.Context, name string) error {
 				// Mask the actual error here as it could be used to enumerate users
 				// within the system.
 				log.Warnf("Failed to delete role: role %v is used by user %v.", name, u.GetName())
-				return trace.BadParameter("failed to delete a role that is still in use by a user, check the system server logs for more details")
+				return errDeleteRoleUser
```

Comment:
I would still `trace.Wrap` it here, because otherwise you'll get a stack trace that points to the runtime's initialization (prior to `func main`).